cmd/govim/config/config.go

```diff
@@ -60,6 +60,12 @@ type Config struct {
 	// Staticcheck enables staticcheck analyses in gopls
 	Staticcheck *bool `json:",omitempty"`
 
+	// CompletionCaseSensitive configures gopls' matching to be case sensitive.
+	// This option only takes effect when both CompletionDeepCompletionsDisable
```

Comment:
"when both CompleteDeepCompletions and CompletionFuzzyMatching are set to false..."

Reply:
Thanks. Fixed